src/SSHDebugPS/AD7/AD7UnixAsyncCommand.cs

```diff
@@ -0,0 +1,122 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Threading;
+using Microsoft.VisualStudio.Debugger.Interop.UnixPortSupplier;
+
+namespace Microsoft.SSHDebugPS
+{
+    internal class AD7UnixAsyncCommand : IDebugUnixShellAsyncCommand
+    {
+        private readonly object _lock = new object();
+        private int _bClosed = 0;
```

Comment:
Why is this an `int`?

Reply:
This was an int before. I didn't change the structure of this command